### PR TITLE
Change popup hotkey to shift-ctrl-M

### PR DESCRIPTION
**Pull request by danfinlay**
Because we want to shift the ctrl of the world.

Per idea by @kevinghim

### CODE REVIEW

**Review by vthomas13 — APPROVED** [review]

**Review by vthomas13 — APPROVED** [review]

File: app/manifest/v3/_base.json [hunk 1]
```diff
@@ -19,10 +19,10 @@
   "commands": {
     "_execute_browser_action": {
       "suggested_key": {
-        "windows": "Alt+Shift+M",
-        "mac": "Alt+Shift+M",
-        "chromeos": "Alt+Shift+M",
-        "linux": "Alt+Shift+M"
+        "windows": "Ctrl+Shift+M",
+        "mac": "Ctrl+Shift+M",
```
Comment (BelfordZ) [inline app/manifest/v3/_base.json:23]:
```suggestion
        "mac": "Command+Shift+M",
```

File: app/manifest/v2/_base.json [hunk 2]
```diff
@@ -20,10 +20,10 @@
   "commands": {
     "_execute_browser_action": {
       "suggested_key": {
-        "windows": "Alt+Shift+M",
-        "mac": "Alt+Shift+M",
-        "chromeos": "Alt+Shift+M",
-        "linux": "Alt+Shift+M"
+        "windows": "Ctrl+Shift+M",
+        "mac": "Ctrl+Shift+M",
```
Comment (BelfordZ) [inline app/manifest/v2/_base.json:24]:
```suggestion
        "mac": "Command+Shift+M",
```

**Comment by ulissesferreira** [comment]
Chrome currently opens Chrome's user profiles when pressing Cmd + Shift + M. Just an FYI regarding conflicts 👍

File: app/manifest/v2/_base.json (hunk 2 above)
Comment (Montoya) [reply app/manifest/v2/_base.json:24]: Echoing what @ulissesferreira said, Chrome on Mac shows user profiles when I do this

File: app/devtools.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
+<!doctype html>
```
Comment (Gudahtt) [inline app/devtools.html:1]: Was this file added by mistake? It's unclear what it does

**Comment by kevinghim** [comment]
Will shift-ctrl-M work on Firefox as well?

**Comment by mcmire** [comment]
How long have we had the existing shortcut? If we've had it for a long time, we may want to keep it. People get _very_ mad (for good reason) when their shortcuts change without notice.

File: app/manifest/v2/_base.json (hunk 2 above)
Comment (HowardBraham) [reply app/manifest/v2/_base.json:24]: Chrome on Windows **also** shows user profiles when I do this.  We can't make this change.

**Pull request closed without merging**